app/controllers/story.controller.js

```diff
@@ -200,3 +200,72 @@ exports.findAllByUser = async (req, res) => {
     }
 }
 
+
+// Add a chat to a story
+
+exports.addChat = async (req, res) => {
+
+    if (req.body.message === undefined) {
+        const error = new Error("Message cannot be empty for chat!");
+        error.statusCode = 400;
+        res.status(400).send({
+            message:
+                error.message || "Some error occurred while adding chat to story.",
+        });
+
+    }
+    const id = req.params.id;
+
+    try {
+        const chat = {
+            role: "User",
+            message: req.body.message,
+            storyId: id,
+        };
+        const history = await Chat.findAll({ where: { storyId: id } });
```

Comment:
finding all "Chats" for a story to include as the history, will the order these Chats come back in matter or ever change?  presumably Cohere (and LLMs, generally) use the history for context but if the history gets sent back out-of-order what will happen?  should your storage of Chats include a timestamp when they were created/added so that we can get them in sorted order?